extensions/sql-bindings/src/sql-bindings.d.ts

```diff
@@ -119,4 +126,12 @@ declare module 'sql-bindings' {
 		azureFunctions: string[];
 	}
 
+	/**
+	 * Result from promptAndUpdateConnectionStringSetting
+	 */
+	export interface IConnectionStringInfo {
+		connectionStringSettingName: string | undefined;
```

Comment:
I don't think it makes sense for this to be `| undefined` - the only places it seems like you're setting it the value is not undefined (and it doesn't really make sense for this to ever be undefined right? We always want this)